telegram/message.py

```diff
@@ -1157,10 +1289,24 @@ def text_markdown_urled(self):
         """
         return self._parse_markdown(self.text, self.parse_entities(), urled=True)
 
+    @property
+    def text_markdown_urled_v2(self):
```

Comment:
Maybe need to use `text_markdown_v2_urled` name, for insight of `v2` relation to `markdown` rather than `urled`?